### PR TITLE
[patch] Code fix for AWS Self Managed and ROSA cluster

### PR DESCRIPTION
**Pull request by shubhammalvankar**
Roles edited:

-   **cp4d_db2wh** : changed the default for a variable
-   **cp4d_install_services** : added a variable with a default value
-   **ocp_verify** : added one more certificate lookup for ROSA ["{{ cluster_name }}-primary-cert-bundle-secret"]
-   **uds_install** : added one more certificate lookup for ROSA ["{{ cluster_name }}-primary-cert-bundle-secret"]

Playbook changed:

-   **configure-app.yml** : added var 'mas_appws_jdbc_binding'

### CODE REVIEW

File: ibm/mas_devops/roles/uds_install/defaults/main.yml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ---
+cluster_name: "{{ lookup('env', 'CLUSTER_NAME')}}"
```
Comment (durera) [inline ibm/mas_devops/roles/uds_install/defaults/main.yml:2]: Please add to the role `README.md` info about this, e.g. that it's required on ROSA, but not used anywhere else

**Review by durera — CHANGES_REQUESTED** [review]

**Comment by shubhammalvankar** [comment]
Added the role variable description in the `README.md` file as requested.

**Review by whitfiea — CHANGES_REQUESTED** [review]

File: ibm/mas_devops/roles/cp4d_db2wh/defaults/main.yml [hunk 2]
```diff
@@ -21,23 +21,23 @@ db2wh_num_pods: "{{ lookup('env', 'DB2WH_NUM_PODS') | default(1, true) }}" # by
 
 # Configure meta storage for db2 (CP4D v3.5 and v4.0)
 db2wh_meta_storage_class: "{{ lookup('env', 'DB2WH_META_STORAGE_CLASS') }}"
-db2wh_meta_storage_size_gb: "{{ lookup('env', 'DB2WH_META_STORAGE_SIZE_GB') | default(20, true) }}"
```
[the remark this answers is not in the document]
Comment (Shrivastava-Varsha) [reply ibm/mas_devops/roles/cp4d_db2wh/defaults/main.yml:24]: @whitfiea the pvc size will be 1MB if we dont specify Gi which is not sufficient. pods were failing to start.

File: multicloud-bootstrap/aws/master-cft/cft-mas-core-dev.json [hunk 3]
```diff
@@ -763,8 +763,8 @@
                                 "# Initiate the provisioning process\n",
                                 "ssh-keyscan github.com >> ~/.ssh/known_hosts;",
                                 "cd /root;",
-                                "git clone -b dev https://github.com/ibm-mas/mas-on-aws.git;",
-                                "cd mas-on-aws;",
+                                "git clone -b validate-bootstrap-edge https://github.com/ibm-mas/ansible-devops.git;",
```
[the remark this answers is not in the document]
Comment (shubhammalvankar) [reply multicloud-bootstrap/aws/master-cft/cft-mas-core-dev.json]: Removed the branch selection in the file.

File: ibm/mas_devops/roles/cp4d_db2wh/defaults/main.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (shubhammalvankar) [reply ibm/mas_devops/roles/cp4d_db2wh/defaults/main.yml:24]: @whitfiea I made similar changes in the defaults/main.yml in the **db2u** role

**Review by whitfiea — DISMISSED** [review]

File: ibm/mas_devops/roles/uds_install/defaults/main.yml (hunk 1 above)
Comment (shubhammalvankar) [reply ibm/mas_devops/roles/uds_install/defaults/main.yml:2]: @durera Made changes to the README.md as requested

**Review by whitfiea — APPROVED** [review]

**Review by durera — APPROVED** [review]

**Pull request merged**